modules/quantcastBidAdapter.js

```diff
@@ -75,6 +75,17 @@ export const spec = {
         });
       });
 
+      let gdprSignal;
+      let gdprConsentString;
+
+      if (bid.gdprConsent !== undefined) {
```

Comment:
How about just check `if (bid.gdprConsent)`, to avoid "Cannot read property of null"?